index.html

```diff
@@ -918,14 +918,19 @@
     </section>
     <section> <h3>Payment scenarios</h3>
       <p>
-        The user buys goods in a store, and payments options include NFC.
+        Payment scenarios with Web NFC generally do not refer to supporting
+        the NFC payment process itself, but associating the payment status with
```

Comment:
Maybe rephrase it differently? NFC can be used as a 'companion' technology for payments, e.g., for EMV, thus, NFC is not a payment process.

> Payment scenarios with Web NFC generally do not refer to supporting the NFC payment process itself

Payment scenarios with Web NFC generally do not refer to supporting the payment process itself